examples/colab/semantic_approximate_nearest_neighbors.ipynb

```diff
@@ -655,7 +655,7 @@
       "source": [
         "## 3. Build the ANN Index for the Embeddings\n",
         "\n",
-        "[ANNOY](https://github.com/spotify/annoy) (Approximate Nearest Neighbors Oh Yeah) is a C++ library with Python bindings to search for points in space that are close to a given query point. It also creates large read-only file-based data structures that are mmapped into memory. It is built and used by [Spotify](https://www.spotify.com) for music recommendations."
```

Comment:
This is not a typo but correct techspeak, see https://en.wikipedia.org/wiki/Mmap. Please leave the text as-is. Feel free to add an explanatory link as seen at https://github.com/spotify/annoy/blob/df5364c9b39526bb7b53e31b1b5fd593e879b58a/README.rst#annoy

Reply:
Got it !